src/app/GitUI/ScriptsEngine/IScriptOptionsProvider.cs

```diff
@@ -11,6 +11,6 @@ public interface IScriptOptionsProvider
     ///  Retrieves the information for placeholders in script arguments.
     /// </summary>
     /// <param name="option">The option identifier which is to be replaced.</param>
-    /// <returns>The value to be used for the script argument.</returns>
-    string? GetValue(string option);
+    /// <returns>The value(s) to be used for the script argument.</returns>
+    IEnumerable<string> GetValue(string option);
```

Comment:
The original contract implied it could return a null, though looking at the implementation it never did. It's worth inspecting all callsites and remove null-handling operations, if any.

Reply:
> It's worth inspecting all callsites and remove null-handling operations, if any.

Former `null` values had already been converted to empty collections.
Since other options can return a single value, `IEnumerable<string>? newStrings` needs to be nullable inside `ParseScriptArguments`.

Reply:
From the API design standpoint `IEnumerable<string>?` is rather undesirable - collection/set return value should never be a null but an empty collection/set.

Reply:
What do you mean, Igor?

![image](https://github.com/user-attachments/assets/75c418a5-a3fe-4dab-9f29-7dd77d6a86f3)

Reply:
I was replying to your earlier comment:

> Since other options can return a single value, IEnumerable<string>? newStrings needs to be nullable inside ParseScriptArguments.

...but appears I have misread it :)